drivers/eks/eks_driver.go

```diff
@@ -35,10 +36,35 @@ import (
 	"k8s.io/client-go/rest"
 )
 
-var amiForRegion = map[string]string{
-	"us-west-2": "ami-0a54c984b9f908c81",
-	"us-east-1": "ami-0440e4f6b9713faf6",
-	"eu-west-1": "ami-0c7a4976cb6fafd3a",
+const (
+	amiNamePrefix = "amazon-eks-node-"
+)
+
+var amiForRegionAndVersion = map[string]map[string]string{
+	"1.11": map[string]string{
+		"us-west-2":      "ami-0a2abab4107669c1b",
+		"us-east-1":      "ami-0c24db5df6badc35a",
+		"us-east-2":      "ami-0c2e8d28b1f854c68",
+		"eu-central-1":   "ami-010caa98bae9a09e2",
+		"eu-north-1":     "ami-06ee67302ab7cf838",
+		"eu-west-1":      "ami-01e08d22b9439c15a",
+		"ap-northeast-1": "ami-0f0e8066383e7a2cb",
+		"ap-northeast-2": "ami-0b7baa90de70f683f",
+		"ap-southeast-1": "ami-019966ed970c18502",
+		"ap-southeast-2": "ami-06ade0abbd8eca425",
+	},
+	"1.10": map[string]string{
```

Comment:
These are not the same as the ones submitted in this pr
https://github.com/rancher/kontainer-engine/pull/108/files
which is right @orangedeng @nathan-jenan-rancher ?

Reply:
The AMIs in that linked PR are not correct.  This PR has the latest AMIs from amazon.  See https://docs.aws.amazon.com/eks/latest/userguide/eks-optimized-ami.html